README.md

```diff
@@ -4,3 +4,5 @@ Test repo for pull requests
 Adding a line at github/jbullock35gmail.
 
 Adding a second line at github/jbullock35gmail. [748am]
+
```

Comment:
From jbullock35gmail, leaving a comment on a range of lines. Choosing "add single comment."